src/PhpSpec/CodeAnalysis/AccessInspectorInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Phpspec\CodeAnalysis;
+
+interface AccessInspectorInterface
```

Comment:
I think let's not drop `*Interface` yet for consistency, but try and make a conscious effort to name the concrete implementations uniquely.